tutorials/finetune_lora.md

```diff
@@ -57,7 +58,7 @@ and optionally with double-quantization:
 python finetune/lora.py --quantize "bnb.nf4-dq"
 ```
 
-The table below lists a comparison with different settings on a StableLM 3B model finetuned with LoRA on Alpaca for 5,000 iterations using a microbatch size of 4:
```

Comment:
Was 5k a mistake? I would expect that changing the # of iters requires updating the training time and loss in the tables

Reply:
This was still the old table, but I just updated it to numbers from the large (new) table.